test/ui-test/testSuites/suite_status/shared/scripts/names.py

```diff
@@ -161,6 +161,15 @@
 createCommunityOutroMessageInput_TextEdit = {"container": statusDesktop_mainWindow_overlay, "objectName": "createCommunityOutroMessageInput", "type": "TextEdit", "visible": True}
 createCommunityNextBtn_StatusButton = {"container": statusDesktop_mainWindow_overlay, "objectName": "createCommunityNextBtn", "type": "StatusButton", "visible": True}
 createCommunityFinalBtn_StatusButton = {"container": statusDesktop_mainWindow_overlay, "objectName": "createCommunityFinalBtn", "type": "StatusButton", "visible": True}
-mainWindow_communityHeader_StatusChatInfoButton = {"container": statusDesktop_mainWindow, "id": "communityHeader", "type": "StatusChatInfoButton", "unnamed": 1, "visible": True}
+mainWindow_createChannelOrCategoryBtn_StatusBaseText = {"container": statusDesktop_mainWindow, "objectName": "createChannelOrCategoryBtn", "type": "StatusBaseText", "visible": True}
+create_channel_StatusMenuItemDelegate = {"checkable": False, "container": statusDesktop_mainWindow_overlay, "enabled": True, "objectName": "createCommunityChannelBtn", "type": "StatusMenuItemDelegate", "visible": True}
+create_category_StatusMenuItemDelegate = {"checkable": False, "container": statusDesktop_mainWindow_overlay, "enabled": True, "objectName": "createCommunityCategoryBtn", "type": "StatusMenuItemDelegate", "visible": True}
+createCommunityChannelNameInput_TextEdit = {"container": statusDesktop_mainWindow_overlay, "objectName": "createCommunityChannelNameInput", "type": "TextEdit", "visible": True}
+createCommunityChannelDescriptionInput_TextEdit = {"container": statusDesktop_mainWindow_overlay, "objectName": "createCommunityChannelDescriptionInput", "type": "TextEdit", "visible": True}
+createCommunityChannelBtn_StatusButton = {"container": statusDesktop_mainWindow_overlay, "objectName": "createCommunityChannelBtn", "type": "StatusButton", "visible": True}
+channel_Header_chat_title_StatusBaseText = {"container": statusDesktop_mainWindow, "objectName": "chatInfoNameText", "type": "StatusBaseText", "visible": True}
+mainWindow_communityHeader_StatusChatInfoButton = {"container": statusDesktop_mainWindow, "objectName": "communityHeaderButton", "type": "StatusChatInfoButton", "visible": True}
 community_ChatInfo_Name_Text = {"container": mainWindow_communityHeader_StatusChatInfoButton, "objectName": "statusChatInfoButtonNameText", "type": "StatusBaseText", "visible": True}
-
+mainWindow_chatLogView_ListView = {"container": statusDesktop_mainWindow, "id": "chatLogView", "occurrence": 2, "type": "ListView", "unnamed": 1, "visible": True}
```

Comment:
`id` don't work in my environment!
no idea why but I don't see id and occurence

Can we use something else?

Reply:
I was able to remove the id and use objectName instead, but if I remove the occurrence , it picks the wrong object sadly. Do you know of a better way to do? I know that occurrence is not the greatest, but I don't know what to do otherwise.

Reply:
I think the problem is because of my previous "hack" in the main codebase, where instead of hiding the channels when they are not longer active, I set the `height` to 0. That was done to make the channel switching faster. It's possible that my latest improvement to channel switching made that "hack" obsolete. I'll test to see if that's indeed the problem. Squish would probably have an easier time if `visible` was indeed `false` on inactive channels.

Reply:
Good news, it did fix it 🙌 
I didn't remove the full hack yet, I'll create an issue to investigate if my change here broke the time improvement and if the hack is still needed